assets/src/modules/map.js

```diff
@@ -426,6 +427,19 @@ export default class map extends olMap {
                     // maxZoom: 19
                     }),
                 });
+            } else if (baseLayerState.type === BaseLayerTypes.Google) {
+                baseLayer = new TileLayer({
+                    minResolution: layerMinResolution,
+                    maxResolution: layerMaxResolution,
+                    preload: Infinity,
+                    source: new Google({
+                        key: baseLayerState.key,
+                        mapType: baseLayerState.mapType,
+                    // use maxZoom 19 to see stretched tiles instead of the BingMaps
```

Comment:
Bingmaps ?
Maybe not directly linked to your PR, but when I see your comment, I'm wondering what happen when in zoom in too much, below what Google is providing ? Does it become blank or pixelated satellite imagery ?

Reply:
> Bingmaps ?

typo, sorry

> I'm wondering what happen when in zoom in too much

I try to load the project using  the `Use native scales according to the project CRS (better display)` option in the Lizmap plugin with min scale = 1, the min displaying scale provided by google in EPSG:3857 is 1:141.

![Screenshot_20240415_121220](https://github.com/3liz/lizmap-web-client/assets/20710470/4351b770-d29f-4e6e-ac10-36ba4c6803fb)

Under this scale the map becomes pixelated.